pkg/arvo/app/sane.hoon

```diff
@@ -0,0 +1,288 @@
+::  %sane: sanity checker for the landscape suite of applications
+::
+::    Userspace currently uses certain identifiers as foreign keys, and
+::    expects those foreign keys to exist in a number of locations. 
+::
+::    These foreign key relationships are prone to breaking during OTAs 
+::    and there are enough of them that they rarely get tested for 
+::    manually. %sane is a gall app that will check the validity of
+::    these relationships. The MVP will just print the errors as it 
+::    finds them.
+::
+::    Pokes:
+::      %off - turn off checking on a reload
+::      %lax - check on reload, printing errors
+::      %strict - check on reload, aborting commit if invalid
+::      %check-print  - run a check now, printing errors
+::      %check-crash - run a check now, crashing nondeterministically
+::        if invalid 
+::    
+::    Currently validates:
+::      - Entries in {contact,metadata,group} stores are in sync with 
+::         their hooks
+::      - Each group has its associated metadata and contacts
+::
+::    Future validation: 
+::      - Subscriptions are correctly setup for hooks
+::      - Graph store integration
+::    
+::
+
+::
+/-  *metadata-store, contacts=contact-store, *group
+/+  default-agent, verb, dbug, resource
+~%  %sane-app  ..is  ~
+|%
++$  card  card:agent:gall
+::
```

Comment:
No need for `::` in between these lines of `+$` unless one of the declarations spans multiple lines.